content/docs/rfcs/10/README.md

```diff
@@ -133,6 +133,12 @@ For an experimental privacy-preserving economic spam protection mechanism, see [
 
 See [23/WAKU2-TOPICS](/spec/23) for more information about recommended topic usage.
 
+# Transport Protocol Specification
+
+This section describes the specification for transport protocols supported by waku v2. While Waku strives to be transport agonistic, however any underlying transport layer MUST be a reliable transport.
```

Comment:
```suggestion
This section describes the specification for transport protocols supported by Waku v2. While Waku v2 strives to be transport agonistic, however any underlying transport layer MUST be a reliable transport.
```

Reply:
```suggestion
Waku v2 is built in top of libp2p, and like libp2p it strives to be transport agnostic.
We define a set of recommended transports in order to achieve a baseline of interoperability between clients.
This section describes these recommended transports.
```

Reply:
- Soften requirement on reliable transport as that's not necessarily set in stone and limits us needlessly
- Reword and be more clear on rationale

Reply:
done ea014867f9062d531dd9f1772c99e4b2058f5f80